adagucserverEC/CCairoPlotter.cpp

```diff
@@ -653,6 +660,33 @@ void CCairoPlotter::drawText(int x, int y, double angle, const char *text) {
   _drawFreeTypeText(x, y, w, h, angle, text, true);
 }
 
+void CCairoPlotter::drawStrokedText(int x, int y, double angle, const char *text, const char *fontFile, float fontSize, float strokeWidth, CColor bgcolor, CColor fgcolor) {
+  if (fontFile) {
+  }
+  /* https://www.cairographics.org/samples/text/ */
+  cairo_save(cr);
+  cairo_path_t *cp = cairo_copy_path(cr);
```

Comment:
Add a comment to explain this is done to prepend the new elements to the path (or maybe I misunderstood what is going on)...